ssv/spectest/tests/runner/pre_consensus_justifications/valid.go

```diff
@@ -0,0 +1,117 @@
+package pre_consensus_justifications
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv"
+	"github.com/bloxapp/ssv-spec/ssv/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+var decideFirstHeight = func(r ssv.Runner) ssv.Runner {
+	i := qbft.NewInstance(
+		r.GetBaseRunner().QBFTController.GetConfig(),
+		r.GetBaseRunner().QBFTController.Share,
+		r.GetBaseRunner().QBFTController.Identifier,
+		qbft.FirstHeight)
+	i.State.DecidedValue = []byte{1, 2, 3, 4}
+	i.State.Decided = true
+	r.GetBaseRunner().QBFTController.StoredInstances = append(r.GetBaseRunner().QBFTController.StoredInstances, i)
+	return r
+}
```

Comment:
Since this is used in lots of places maybe move this out to testutils?

Reply:
please open an issue for it so we can do it later